README.md

```diff
@@ -1,2 +1,82 @@
 # velero-plugin
-HPE plugin for Velero
```

Comment:
I'm a bit confused here. Isn't this a Velero plugin for CSPs?

"Container Storage Provider ([CSP](https://github.com/hpe-storage/container-storage-provider)) plugin for Velero"?

Reply:
as discussed offline will keep this HPE